01-filedir.md

```diff
@@ -73,29 +91,26 @@ $ pwd
 /Users/nelle
 ~~~
 
-> ## Home directory {.callout}
+> ## Home Directory Variation {.callout}
 >
 > The home directory path will look different on different operating systems.
 > On Linux it will look like `/home/nelle`,
 > and on Windows it will be similar to `C:\Documents and Settings\nelle`.
-> Note that it may look slightly different for different versions of Windows.
+> (Note that it may look slightly different for different versions of Windows.)
+> In future examples, we've used Mac output as the default - Linux and Windows 
+> output may differ slightly, but should be generally similar.  
 
+To understand what a "home directory" is,
+let's have a look at how the file system as a whole is organized.  For the 
+sake of example, we'll be 
+illustrating the filesystem on our scientist Nelle's computer.  After this
+illustration, you'll be learning a command to explore your own filesystem, 
```

Comment:
I'd say commands since they are learning pwd, cd and ls, which all give info about the filesystem but not a high priority change.